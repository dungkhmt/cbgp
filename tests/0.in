10 10
10 23
10 7
2 10
6 1
1 9
1 5
6 10
10 4
1 3
4 3
1 10
6 7
5 6
5 7
10 9
9 2
3 2
3 10
4 2
3 8
2 1
4 8
7 1
7 3
