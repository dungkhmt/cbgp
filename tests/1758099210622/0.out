(9, 6), (1, 9), (5, 0), (0, 8), (0, 4), (5, 9), (9, 3), (0, 2), (3, 2), (0, 9), (5, 6), (4, 5), (4, 6), (9, 8), (8, 1), (2, 1), (2, 9), (3, 1), (2, 7), (1, 0), (3, 7), (6, 0), (6, 2), 
numberIntersection=0 
minAngle=0.03602044942918958 
sumAngle=4.573504721083623 
minEdgeLength=4.0 
sumEdgeLength=364.5043163490399 
minDistance=0.6153846153846154 
sumDistance=2521.0073340199615 

0.0000000000 0.0360204494 4.5735047211 4.0000000000 364.5043163490 0.6153846154 2521.0073340200
0: (20, 12),
1: (26, 8),
2: (2, 4),
3: (32, 0),
4: (14, 20),
5: (14, 24),
6: (10, 16),
7: (0, 0),
8: (22, 16),
9: (16, 32),
20 12 26 8 2 4 32 0 14 20 14 24 10 16 0 0 22 16 16 32 
0: (20, 12), 1: (26, 8), 2: (2, 4), 3: (32, 0), 4: (14, 20), 5: (14, 24), 6: (10, 16), 7: (0, 0), 8: (22, 16), 9: (16, 32), 
numberIntersection=0 minAngle=0.03602044942918958 sumAngle=4.573504721083623 minEdgeLength=4.0 sumEdgeLength=364.5043163490399 minDistance=0.6153846153846154 sumDistance=2521.0073340199615 
Time taken: 24.917 s
